t/unit/worker/test_consumer.py

```diff
@@ -637,6 +637,7 @@ def test_qos_global_worker_detect_quorum_queues_true_with_quorum_queues(self):
     def test_qos_global_eta_warning(self):
```

Comment:
Well, basically, if we enable ETA via native delayed delivery automatically, then we don’t need any warning anymore. The feature is complete.

It’s better not to include extra code “just because it was done”, it increases the maintenance complexity.
Please remove the warning or present a valid argument to maintain it, as I don’t see a reason anymore.

Reply:
Let me be more clear - the warning message itself isn’t needed anymore, not the warning code.
It requires rephrasing that only the global QoS is disabled and how it effect prefetch count, e.g., `Detected quorum queue "%r", disabling global QoS.` and that’s it plus minus.

The ETA is no longer a problem to report the user about, it will just work now.

Reply:
We need the warning since we're keeping the feature flag.

Reply:
> We need the warning since we're keeping the feature flag.

Let’s agree for now that the warning is indeed dependent on this decision.
If the flag remains, so will the warning, and vice versa.